superset-frontend/src/dashboard/components/nativeFilters/FilterBar/FilterControls/FilterValue.tsx

```diff
@@ -270,7 +277,7 @@ const FilterValue: React.FC<FilterProps> = ({
           height={HEIGHT}
           width="100%"
           showOverflow={showOverflow}
-          formData={formData}
+          formData={formDataWithDisplayParams}
```

Comment:
hmmm I'm not sure if orientation and overflow belongs to `formData`... I'd rather add a new optional prop to `SuperChart` like `displaySetting: Record<string, any>` and put those values there.